libraries/timeline-view-compose/src/main/java/com/trendyol/uicomponents/timelineviewcompose/TimelineView.kt

```diff
@@ -0,0 +1,127 @@
+package com.trendyol.uicomponents.timelineviewcompose
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.trendyol.uicomponents.timelineviewcompose.item.HorizontalTimelineImageItem
+import com.trendyol.uicomponents.timelineviewcompose.item.HorizontalTimelineItem
+import com.trendyol.uicomponents.timelineviewcompose.item.VerticalTimelineImageItem
+import com.trendyol.uicomponents.timelineviewcompose.item.VerticalTimelineItem
+import com.trendyol.uicomponents.timelineviewcompose.model.FakeTimelineItemProvider.provideTimelineItemList
+import com.trendyol.uicomponents.timelineviewcompose.model.TimelineItem
+import com.trendyol.uicomponents.timelineviewcompose.model.TimelineOrientation
+
+@Composable
+fun TimelineView(
+    items: List<TimelineItem>,
```

Comment:
🤔 😄  you like list 😸

Reply:
yes, I still don't like it 😄 but if I use ImmutableList here, I'm forcing the immutableCollection lib on those who will use this view.
besides, this view is static, we can ignore it.

Reply:
You're right